src/snowflake/snowpark/modin/plugin/_internal/apply_utils.py

```diff
@@ -95,22 +71,66 @@
 cloudpickle.register_pickle_by_value(sys.modules[__name__])
 
 SUPPORTED_SNOWPARK_PYTHON_FUNCTIONS_IN_APPLY = {
-    exp,
-    ln,
-    log,
-    _log2,
-    _log10,
-    sin,
-    cos,
-    tan,
-    sinh,
-    cosh,
-    tanh,
-    ceil,
-    floor,
-    trunc,
-    sqrt,
-    snowflake_cortex_summarize,
+    sp_func.exp,
+    sp_func.ln,
+    sp_func.log,
+    sp_func._log2,
+    sp_func._log10,
+    sp_func.sin,
+    sp_func.cos,
+    sp_func.tan,
+    sp_func.sinh,
+    sp_func.cosh,
+    sp_func.tanh,
+    sp_func.ceil,
+    sp_func.floor,
+    sp_func.trunc,
+    sp_func.sqrt,
+    sp_func.snowflake_cortex_summarize,
+}
+
+NUMPY_FUNCTION_TO_SNOWFLAKE_FUNCTION = {
```

Comment:
The `numpy_to_pandas` file should changed as well, right? The numpy ufunc function mapping should be in one place.

Reply:
They are coming from different sources. Let me see if it's possible to consolidate this into one.

Reply:
one thing we can do is change entries in numpy_to_pandas.numpy_to_pandas_universal_func_map something like this
`"exp": lambda obj, inputs: obj.apply(sp_func.exp)` -> `"exp": lambda obj, inputs: obj.apply(np.exp)` 
And then numpy to snowpark function mapping will only be in `snowflake_query_compiler.applymap`
Does it make sense or were you thinking something else? @sfc-gh-jkew

Reply:
That might add an unnecessary level of indirection in some cases.  
   `df.apply(np.cos)` is the same as `np.cos(df)`and ultimately it should all map to `sp_func.cos`

The np.<function> mapping can be occasionally different than the df.apply(<function>) mapping, but for simple unary functions it's probably the same. We might be able to refactor this to treat unary functions differently, but that might be to confusing to the reader.

Maybe the best idea is to move `NUMPY_FUNCTION_TO_SNOWFLAKE_FUNCTION` into `numpy_to_pandas.py` - at least then we have all numpy compat stuff in the same file.

Reply:
Moved `NUMPY_FUNCTION_TO_SNOWFLAKE_FUNCTION` into `numpy_to_pandas.py`